### PR TITLE
test: lib/bzero: Added unit tests for lib/bzero.

### PR DESCRIPTION
**Pull request by mwierzbix**

### CODE REVIEW

File: test/cmocka/src/lib/lib/bzero.c [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+ * Author: Michal Jerzy Wierzbicki <michalx.wierzbicki@linux.intel.com>
+ */
+
+#include <sof/alloc.h>
+
+#include <stdarg.h>
+#include <setjmp.h>
+#include <cmocka.h>
+
+static int checkArr(char *arr, size_t n, char shouldBe)
```
Comment (jajanusz) [inline test/cmocka/src/lib/lib/bzero.c]: Please do not mix camelCase with snake_case.

File: test/cmocka/src/lib/lib/bzero.c [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+{
+	size_t i;
+
+	for (i = 0; i < n; ++i)
+		if (arr[i] != shouldBe)
+			return i;
+	return -1;
+}
+
+static void test_lib_lib_bzero_checkArr(void **state)
```
Comment (jajanusz) [inline test/cmocka/src/lib/lib/bzero.c]: Please do not mix camelCase with snake_case.

File: test/cmocka/src/lib/lib/bzero.c [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+{
+	//            aa000aa - pattern of zeroing
+	char str[] = "aaaaaaa";
+	const int size = sizeof(str)-1;
+	const int zeroBegin = 2;
+	const int zeroEnd = 4;
+
+	(void) state;
+	bzero(str+zeroBegin, zeroEnd-zeroBegin);
+	assert_int_equal(checkArr(
```
Comment (jajanusz) [inline test/cmocka/src/lib/lib/bzero.c]: Please, can you split it into 3 unit tests so we have 1 assert per test?

Comment (mwierzbix) [reply test/cmocka/src/lib/lib/bzero.c]: Ok, will split into 3 uts with same data/logic but different checks.

File: test/cmocka/src/lib/lib/bzero.c (hunk 2 above)
Comment (mwierzbix) [reply test/cmocka/src/lib/lib/bzero.c]: Will fix, and remember for future comitsssss.

**Comment by mwierzbix** [comment]
@jajanusz Done.

**Comment by jajanusz** [comment]
Code is ok, but please rebase+squash instead of merging to just have 1 commit with your unit test in pull request.

**Comment by mwierzbix** [comment]
@jajanusz Squashed commits/merges as per request.

**Pull request merged**